spec/support/shared_contexts/mailer.rb

```diff
@@ -1,14 +0,0 @@
-# frozen_string_literal: true
-
-shared_context 'mailer' do
-  before do
-    Sidekiq::Testing.inline!
```

Comment:
this seems to be the online line you need to get mail to work if you have more proactive defaults (i.e. what I added in the rails_helper)

Reply:
I supposed "online" here was supposed to be "only". Also, did you mean that we need to keep that line somewhere, or should add it on a case by case basis?

Reply:
yep I meant only. since it is the only line here we really need to keep to make mail work in the test suite (or just drain the activejob worker) IMO it should be done on a case by case basis. if you are testing a delayed action and you want it to be inline this line can be added, otherwise it's safe to assume jobs are queued via the `fake!` strategy. this makes mail consistent with every other delayed action so we don't need to treat it as a special case via a shared context or helper method, if that makes sense

Reply:
Perfect, I'll go ahead and merge it then!